File: Data Structures & Algorithms/search-2d-matrix/submission-0.cs
public class Solution {
    public bool SearchMatrix(int[][] matrix, int target) {
        for(int row=0;row<matrix.Length;row++){
            for(int col=0;col<matrix[row].Length;col++){
                if(matrix[row][col]==target){
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/valid-sudoku/submission-0.java
class Solution {
    public boolean isValidSudoku(char[][] board) {
        Set<String> seen = new HashSet<>();
        for(int r = 0; r < 9; r++){
            for(int c = 0; c <9 ; c++){
                char val = board[r][c];
                if(val != '.'){
                    String rowKey = "row" + r + val;
                    String colKey = "col" + c + val;
                    String boxKey = "box" + (r/3) + (c/3) + val;

                   if (!seen.add(rowKey) || !seen.add(colKey) || !seen.add(boxKey)){
                    return false;
                   }
                }   
            }
        }
        return true;
    }
}
